docs/source/contribution.rst

```diff
@@ -0,0 +1,14 @@
+Contribution_Guidelines
+=======================
+
+To contribute to JetPack the skills required are:
+
+  * Familiar with Ansible
+  * Good understanding and debugging of `infrared <https://github.com/redhat-openstack/infrared>`_  ansible tasks
+  * Willigness to debug the undercloud and overcloud deployment failures :)
+
+References
+----------
+
+https://docs.google.com/document/d/1qtmjUNoKvKvVFA0QYoK7OD1tC_Q5L-qkjMoSsNSrtWI/edit?usp=sharing
```

Comment:
Remove references section. These are our internal docs and shouldn't be shared for others.

Reply:
@venkataanil Have addressed your comments. Please have a look!